pyiron_gui/project/project_browser.py

```diff
@@ -99,13 +105,138 @@ def __repr__(self):
 
     def self_representation(self):
         """Self representation of the wrapped object if known, else None is returned."""
-        if self._type is None:
-            return
-        if self._type == "structure":
-            return self._wrapped_object.plot3d()
-        if self._type == 'murnaghan':
-            plt.ioff()
-            self._wrapped_object.plot()
+        return None
+
+
+class AtomsWrapper(BaseWrapper):
+    def __init__(self, pyi_obj, project, rel_path=""):
+        super().__init__(pyi_obj, project, rel_path=rel_path)
+        self._type = 'structure'
+        self._ngl_widget = None
+        self._options = {
```

Comment:
Does it make sense to package this data structure up as its own class? My main motivation is that 'refactor:rename' will be a pain in the butt in the future if we're looking for string instances of the field names, but could be easier if we're looking for particular attributes of an `_Options` instance.

Reply:
It might be overkill right now, I just wanted to float the idea.